API/SSW.Rewards.Application/Reward/Queries/GetRecentRewards/GetRecentRewardsQuery.cs

```diff
@@ -0,0 +1,10 @@
+﻿using MediatR;
+using System;
+
+namespace SSW.Rewards.Application.Reward.Queries.GetRecentRewards
+{
+    public class GetRecentRewardsQuery : IRequest<RecentRewardListViewModel>
```

Comment:
You can put the Query and the handler in the same file. 

Makes it easier to find the matching Handler when you look at references for the Query.

Reply:
Hi Will, in the most recent Clean Architecture template the handler classes are in separate files so I was just following this style. Is this a matter of stylistic preference or should we have a definitive (SSW Rule?) guide to this?

Reply:
No i don't think there is/should be a rule for this, it's just preference. Also some people may complain further and say 1 class 1 file. But i just find it a lot more convenient having them together. Or even having the Handler as a nested class in the Query. Again, just preference.

No change required unless you find it useful. :)